bcgov/biohubbc
# SIMSBIOHUB-287: User Access Request

## Links to Jira Tickets

[SIMSBIOHUB-287](https://apps.nrs.gov.bc.ca/int/jira/browse/SIMSBIOHUB-287)

## Description of Changes

- Updated create access request to place email and display name in the correct fields
- Updated dev users that were affected by issue

## Testing Notes

- Create a test BCeID
- Login to SIMS with your test BCeID and request access
- Login as admin user and approve the request
- Edit a project
- Open the Manage Team members autocomplete and see that the new user display name and email are consistent with other seed users
